### PR TITLE
Docker file

### PR DESCRIPTION
**Pull request by chebroluharika**
**Description**
Create DockerFile

**Issues Resolved**
NA

### Check List
- [ ] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [ ] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## v5.11.0(unreleased)
```
Comment (soodpr) [inline CHANGELOG.md:1]: i think your release version should be 5.10.1 as its just a patch

Comment (soodpr) [reply CHANGELOG.md:1]: ignore this as another PR is extending the support of Hypervsior Manager for API 1000/1200

Comment (chebroluharika) [reply CHANGELOG.md:1]: I believe, I should not do any change based on above comments. Please correct me if I am wrong.

Comment (soodpr) [reply CHANGELOG.md:1]: thats correct

**Pull request merged**